### PR TITLE
Add filter by property_uuid in articles list (pageviews and conversions)

### PR DESCRIPTION
**Pull request by megadevel**
Problem: In pageview stats and conversion stats displayed all articles (Selected property in left sidebar doesn't affect it). It is problem if you use remp for few different sites.

### CODE REVIEW

File: Beam/app/Http/Controllers/ArticleController.php [hunk 1]
```diff
@@ -32,11 +33,15 @@ class ArticleController extends Controller
 
     private $conversionRateConfig;
 
-    public function __construct(JournalContract $journal, ConversionRateConfig $conversionRateConfig)
+    private $tokenProvider;
+
+    public function __construct(JournalContract $journal, ConversionRateConfig $conversionRateConfig, TokenProvider $tokenProvider =  null)
```
Comment (rootpd) [inline Beam/app/Http/Controllers/ArticleController.php]: Is there a reason why the `$tokenProvider` is made optional? The initialization in `app/Providers/RempJournalServiceProvider.php` is done in a way that there always should be an implementation.

If not, I'd make it required and remove `if ($this->tokenProvider) {` conditions below.

Comment (megadevel) [reply Beam/app/Http/Controllers/ArticleController.php]: fixed

Comment (rootpd) [reply Beam/app/Http/Controllers/ArticleController.php]: Thanks. Will you please also squash and force push the commits? We'll be merging this in our internal repository which gets mirrored here and if we squash it there, Github won't be able to match your commits with our squash.

When it's squashed, we'll merge this to master.

Comment (megadevel) [reply Beam/app/Http/Controllers/ArticleController.php]: OK, I got it.

**Pull request merged**